airflow/jobs.py

```diff
@@ -1143,13 +1145,16 @@ def _find_executable_task_instances(self, simple_dag_bag, states, session=None):
             # DAG IDs with running tasks that equal the concurrency limit of the dag
             dag_id_to_possibly_running_task_count = {}
 
-            for task_instance in priority_sorted_task_instances:
+            # Number of tasks that cannot be scheduled because of no open slot in pool
+            num_starved_tasks = 0
+            for i, task_instance in enumerate(priority_sorted_task_instances):
```

Comment:
Can we make this variable name a bit more meaningful?

Reply:
@Fokko which variable are you referring to? I assume it is i.